dev/mux/Makefile

```diff
@@ -1,19 +1,39 @@
-VCS             :=  vcs
+SHELL           := /bin/bash
```

Comment:
The `|&` in highlight is `bash` specific syntax, add this line to let make know we're using `bash`. Default for Ubuntu is `sh`.